clients/go/coreutils/literals.go

```
@@ -483,11 +483,11 @@ func MakeLiteralForType(t *core.LiteralType, v interface{}) (*core.Literal, erro
 		}
 	case *core.LiteralType_Simple:
```

can we add a default here?

---

Like the default should raise an error, saying unsupported type. 
Also how do we handle Schema type and Blob type?